scalafix-core/shared/src/main/scala/scalafix/internal/rule/Sbt1.scala

```diff
@@ -7,7 +7,11 @@ import scalafix._
 import scalafix.rule.RuleCtx
 import scalafix.rule.SemanticRule
 
-case class Sbt1(index: SemanticdbIndex) extends SemanticRule(index, "Sbt1") {
+case class Sbt1(index: SemanticdbIndex)
+    extends SemanticRule(
+      index,
+      "Sbt1",
+      "Semantic rule to support sbt version 1 changes in syntax.") {
```

Comment:
Rewrite to convert sbt 0.12 deprecated operators like `<++=` into sbt 0.13 `:=`/`.value` syntax